front/src/common/BootstrapSNCF/NavBarSNCF.tsx

```diff
@@ -47,11 +47,11 @@ const LegacyNavBarSNCF = ({ appName, showLogoWithName }: Props) => {
   }, [deploymentSettings, showLogoWithName]);
 
   const userDropdownTitle = (
-    <div className={cx({ 'impersonated-user': impersonatedUser })}>
+    <div className={cx('user-dropdown', { 'impersonated-user': impersonatedUser })}>
       {impersonatedUser ? (
-        <Hubot size="lg" className="mr-xl-2" />
+        <Hubot size="sm" className="mr-xl-2" />
       ) : (
-        <Person variant="fill" size="lg" className="mr-xl-2" />
+        <Person variant="base" size="sm" className="mr-xl-2" />
```

Comment:
`base` is already the default `variant`
```suggestion
        <Person size="sm" className="mr-xl-2" />
```

Reply:
Done